CONTRIBUTING.md

```diff
@@ -143,10 +143,6 @@ Before merging a pull request, please make sure the following requirements are m
 
 We provide preview deployments for pull requests. You can view the preview deployment by clicking the "Preview" link in the "View Deployment" section.
 
-### Documentation
-
-If you are adding a new feature, please make sure to update the documentation accordingly. Refer to our [documentation repository](https://github.com/dream-num/docs) for more guidance.
-
```

Comment:
The original link was 404, so I removed the section.

Reply:
Plesae change it to https://github.com/dream-num/univer.ai

Reply:
Still 404, is it because the repository is not public?

Reply:
Sorry. My mistake.